shared/constants/types/teams.tsx

```diff
@@ -121,6 +121,17 @@ export type TeamDetails = {
   requests?: Set<string>
 }
 
+export type RolePair = {
+  role: MaybeTeamRoleType
+  implicitRole: MaybeTeamRoleType
```

Comment:
Could this be a boolean?

Reply:
It could be owner or admin, i thought i'd plumb it all the way through.

Reply:
You're right, it should be a boolean.